src/java/native/zlogjni.cc

```diff
@@ -11,6 +11,8 @@ void Java_org_cruzdb_zlog_Log_disposeInternal(
   delete reinterpret_cast<zlog::Log*>(jhandle);
 }
 
+zlog::Options options;
```

Comment:
This definitely looks like a nasty bug waiting to happen. Why move options out of the stack allocation?